src/writing-your-own-application/generate-project/index.md

```diff
@@ -0,0 +1,52 @@
+# Generating Projects from Templates
+
+We currently maintain two template repositories:
+- [esp-template] - `no_std` template.
+- [esp-idf-template] - `std` template.
```

Comment:
Here we give both template names and project types (std, no_std). The subsections overview the templates, then the following two sections overview applications based on project types.

We switch back and forth between template names and project types which is a bit disorienting.

I suggest that we either stick with esp-template/esp-idf-template or std/no_std apps.